app/views/dashboard/autoyast.xml.erb

```diff
@@ -9,7 +9,8 @@
         <interpreter>shell</interpreter>
         <source><![CDATA[
         #!/bin/sh
-        echo "master: <%= @controller_node %>" > /etc/salt/minion.d/master.conf
+        echo "master: <%= @controller_node %>" > /etc/salt/minion.d/minion.conf
```

Comment:
minion.conf is used on the admin node... So this at least makes them consistent. Lets change them later, as this is blocking update testing etc